akka-actor/src/main/scala/akka/util/Unused.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util
+
+import akka.annotation.InternalApi
+
+/**
+ * Marker for explicit or implicit parameter known to be unused, yet
+ * still necessary from a binary compatibility perspective
+ * or other reason. Useful in combination with
```

Comment:
Another case where this annotation is useful when the parameter is not used in the class where the method is defined, but is used in a subclass that overrides this method. Perhaps worth mentioning, perhaps too much detail :)

Reply:
Oh @raboof I didn't get to this one, WDYT, needed at this time?